src/test/resources/features/RFMGRSflowPages.feature

```diff
@@ -16,26 +16,164 @@ Feature: RFM ultimate parent entity and New nominated Filling Member GRS journey
     And I click on Continue button
     Then I should be on RFM CYA Page
     When I click on Save&Continue button
-    And I access RFM corporate position page
+    Then I should be on RFM Saving Progress Page
+    When I click on Continue button
+    Then I should be on RFM Corp Position Page
+    And The Heading should be What is your position in the corporate structure of the group?
+    When I click on Continue button
+    Then I should see an error message Select if you are the ultimate parent entity or a new nominated filing member on the RFM journey error Page
     When I select corp position as NFM
     And I click on Continue button
     Then I should be on New NFM guidance page
     When I click on Continue button
     Then I should be on RFM registered in UK page
+    And The caption is Group details
+    And The Heading should be Is the new nominated filing member registered in the UK?
+    And I click on Continue button
```

Comment:
please check with one of the devs , whether this content is covered in unit testing and get rid of content testing in ATs